scripts/puppeteer.server.js

```diff
@@ -111,7 +138,25 @@ document.getElementById('actions').addEventListener('click', async (event) => {
         break;
     }
   } catch (error) {
-    // eslint-disable-next-line no-console
+    console.error('Unable to perform an action:', error);
+  }
+});
+
+document.getElementById('members').addEventListener('click', async (event) => {
+  event.preventDefault();
+
+  try {
+    switch (event.target.id) {
+      case 'get-members':
+        console.log(webexSDKAdapter.membershipsAdapter);
+        await webexSDKAdapter.membershipsAdapter
```

Comment:
Is the `await` necessary? isn't this an observable?
```suggestion
       webexSDKAdapter.membershipsAdapter
```